docs/data-collection/emergency-procedures.md

```diff
@@ -6,7 +6,8 @@
 
     !!! danger "Cutting electricity will NOT turn off the magnetic field!"
 
-- [ ] Remove the subject from the scanner's room, applying the [manual extraction procedure described below](#manual-extraction-procedure).
+- [ ] Remove the subject from the scanner's room.
+If the bed is stuck inside the bore apply the first step of [manual extraction procedure described below](#manual-extraction-procedure) and help the participant leave the room.
```

Comment:
This change requires vetting from @celprov and @helenelajous. I'm not even sure I totally understand.

That said, I guess you meant to indent the second sentence:

```suggestion
- [ ] Remove the subject from the scanner's room.
    If the bed is stuck inside the bore, apply the first step of [manual extraction procedure described below](#manual-extraction-procedure) and help the participant leave the room.
```

The problem for me is what *If the bed is stuck inside the bore* actually means. What is in the mindset of anticipating such possibility.

Reply:
I posted this in an issue #148